llvm/include/llvm/MC/MCAsmBackend.h

```diff
@@ -199,7 +199,7 @@ class MCAsmBackend {
   // Defined by linker relaxation targets to possibly emit LEB128 relocations
   // and set Value at the relocated location.
   virtual bool relaxLEB128(MCLEBFragment &LF, MCAsmLayout &Layout,
-                           int64_t &Value) const {
+                           int64_t &Value, bool &UseZeroPad) const {
```

Comment:
Instead of an output parameter, make it part of the return value `pair<bool, bool>`.

The current code does not initialize `UseZeroPad` in all cases.